src/commands/build/internal/getLaunchPageCode.ts

```diff
@@ -22,7 +22,7 @@ data _null_;
    else url="&systcpiphostname";
  end;
  else url="&systcpiphostname";
- url=cats(url,"/SASJobExecution?_FILE=&appLoc/services/");
+ url=cats(url,"/SASJobExecution?_FILE=${appLoc}/services/");
```

Comment:
same issue here (the appLoc needs to be resolved in SAS, so that it can be a parameter at the start of the build.sas program)

Reply:
fixed